### PR TITLE
docs/guide/helper-html.md small fixes [ci skip]

### PR DESCRIPTION
**Pull request by softark**

### CODE REVIEW

File: docs/guide/helper-html.md [hunk 1]
```diff
@@ -92,21 +93,23 @@ if ($type === 'success') {
     Html::addCssClass($options, ['btn-success', 'btn-lg']);
 }
 
-echo Html::tag('div', 'Save', $options); // renders `class="btn btn-success btn-lg"`
+echo Html::tag('div', 'Save', $options);
+// renders '<div class="btn btn-success btn-lg">Save</div>'
 ```
 
-`Html::addCssClass()` prevents duplicating classes, so you don't need to worry the same class may appear twice:
+`Html::addCssClass()` prevents duplicating classes, so you don't need to worry if the same class may appear twice:
```
Comment (klimov-paul) [inline docs/guide/helper-html.md]: Should not coma be here: before 'if'?

Comment (softark) [inline docs/guide/helper-html.md]: 'worry if ...' is almost equivalent to 'worry about the possibility that ...'.
So you don't need a comma before 'if'.

Comment (softark) [inline docs/guide/helper-html.md]: Um, wait a moment, please.
I might be wrong.

Comment (klimov-paul) [inline docs/guide/helper-html.md]: There are 2 senetnces: "you don't need..." and "class may appear" with conjunction. As far as I know it should be a comma between them.

**Comment by klimov-paul** [comment]
Thank you

**Pull request merged**